### ISSUE TITLE
It's a BUG in PullToRefreshGridView?

### ISSUE THREAD

**Issue by asker517** [naver/android-pull-to-refresh#22]
early, i found the issue in old pulltorefreshGridview...but now still exists..

just do like this :  
   everytime you pull down or pull up to load more ..just make the new data size is only 1..that meas gridview only add one item at a time .. when you gridview items over the current screen, you pull up again,you will found the new item will not display...and go on you pull up again,the gridview display 2 new items..
  i don't know is that a bug ?  but when you load data size is > 1 at a time ,it seems display normal.

for help, thank you.

**Comment by ncoolz** [comment]
Yes, this issue is surely a bug. I will try to find the reason and fix it.

**Comment by asker517** [comment]
@ncoolz  fixed or not yet?

**Comment by ncoolz** [comment]
Sorry, it's no time to work on it, I'll try to start as soon as possible.

**Comment by asker517** [comment]
thanks for you reply, so kind of u... :)

------------------ 原始邮件 ------------------
发件人: "Wonjun Kim"notifications@github.com; 
发送时间: 2014年10月10日(星期五) 晚上10:42
收件人: "naver/android-pull-to-refresh"android-pull-to-refresh@noreply.github.com; 
抄送: "Anonymoμs "6781219@qq.com; 
主题: Re: [android-pull-to-refresh] It's a BUG in PullToRefreshGridView?(#22)

Sorry, it's no time to work on it, I'll try to start as soon as possible.

**Comment by zalyulute** [comment]
Seems fixed if correct class PullToRefreshAdapterViewBase > method isLastItemVisible() > "if (lastVisiblePosition >= lastItemPosition - 1)" to "if (lastVisiblePosition >= lastItemPosition)".

Any concern on this correction?

**Comment by ncoolz** [comment]
@zalyulute Yes, you're right. Thanks for resolving this issue. A new commit for this issue will be pushed.

**Comment by asker517** [comment]
@zalyulute @ncoolz It seems solved the bug..but how it did it ? ...I can not find the reason...please make a simple explanation ..thank you very much.

**Comment by cctravel** [comment]
wooo.it's hard to resolving this issue
